homeworks/nazar_kostiv/HW04/tests.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import argparse
+from pyats.aetest import Testcase, test, main, loop, setup
+
+mapping = {
+    'a': (1, 3, 4, 5, 6, 7, 8),
+    'b': (0, 2, 3, 4),
+    'c': (7, 9, 0, 6, 5, 4, 3, 1)
+}
+
+parser = argparse.ArgumentParser(description="Parsing letter")
+parser.add_argument('--letter', required=True, type=str)
+args, sys.argv[1:] = parser.parse_known_args(sys.argv[1:])
+
+class Test(Testcase):
+
+    @setup
+    def setup(self, letter=args.letter):
```

Comment:
@ProffesorRabbit `letter` should not have a default value.

Reply:
@extsoft fixed